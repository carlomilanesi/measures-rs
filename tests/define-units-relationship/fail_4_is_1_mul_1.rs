measures::define_measure_types! {
    exact,
    vector_properties [
        P1 [ U1 { suffix: " u1" } ]
        P2 [ U2 { suffix: " u2" } ]
        P3 [ U3 { suffix: " u3" } ]
    ]
    relationships [ U1 4 == U2 1 * U3 1 ]
}

fn main() {}
