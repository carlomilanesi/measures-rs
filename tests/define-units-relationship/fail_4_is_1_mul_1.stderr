error: no rules expected `4`
  --> tests/define-units-relationship/fail_4_is_1_mul_1.rs:23:12
   |
23 |         U1 4 == U2 1 * U3 1,
   |            ^ no rules expected this token in macro call
   |
note: while trying to match `1`
  --> src/define_units_relationship.rs
   |
   |     { $exact:tt $with_approx:tt $with_correlation:tt, $unit1:ident 1 == $unit2:ident 1 * __ 1 } => {
   |                                                                    ^
